File: 219_Contains_Duplicate_II.js
var containsNearbyDuplicate = function(nums, k) {
    let i = 0;
    while(i < nums.length - 1){
        let t = nums[i];
        let j = i + 1;
        while(j < nums.length){
            if(t === nums[j] && Math.abs(i - j) <= k) return true; 
            j++;
        }
        i++;
    }
    return false;
};

File: 219_Contains_Duplicate_II.cpp
class Solution {
public:
    bool containsNearbyDuplicate(vector<int>& nums, int k) {
       int i,j;
       for(i = 0; i < nums.size() - 1; i++){
           j = i + 1;
           while(j < nums.size() && j <= i + k){
               if(nums[i] == nums[j]) return true;
               j++;
           }
       } 
       return false;
    }
};
